steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -83,35 +120,47 @@ class EvidenceUpload extends Question {
                 uploadEv: b.documents[0].originalDocumentName,
                 link: b.documents[0]._links.self.href
               };
+
               return fs.unlink(pathToFile, next);
             }
             return next(forwardingError);
           });
         });
-      }
-      return next();
-    });
+      });
+    }
```

Comment:
Anyway this can be broken up? It's quite big.

Reply:
I plan to, but possibly in my next pr, if that is ok. This baby has a long journey to go still before arriving anywhere near prod.